torch/_torch_docs.py

```diff
@@ -2740,20 +2740,26 @@ def merge_dicts(*dicts):
     tensor([-1.,  1., -1., -1.])
 """.format(**common_args))
 
-add_docstr(torch.floor_divide,
-           r"""
+add_docstr(torch.floor_divide, r"""
 floor_divide(input, other, *, out=None) -> Tensor
 
-Return the division of the inputs rounded down to the nearest integer. See :func:`torch.div`
-for type promotion and broadcasting rules.
+.. warning::
+    This function's name is a misnomer. It actually rounds the
```

Comment:
can we say something like "we plan to address in a future release" (you can give any level of specificity you want to the plan), it just...kind of looks like we are okay with this.

Reply:
I was also thinking about that. Let's do it.